### DISCUSSION TITLE
reflected payload in html

### DISCUSSION THREAD

**Discussion by JadGh7** [hahwul/dalfox#555] [Q&A]
can someone please explain what does this mean:
W] Reflected Payload in HTML: title='>asd
    1 line:  7O3Yp">Biryani Needs'>asd</div></div></div></div><section class="container d-fle
[POC][R][GET][inHTML-URL]

**Comment by hahwul** [comment]
@JadGh7 
I'm sorry for the late response. 

This means that the user's input was reflected in the HTML without the special characters being escaped. 

By default, this is an HTML Injection. If you can find a way to execute Javascript from here, it could evolve into an XSS (Cross-Site Scripting) attack, and if you can control CSS, it could become a CSS Injection :)


* [POC]: This PoC
* [R]: Payload reflected
* [GET]: HTTP Method
* [inHTML-URL]: Inserted in the HTML area, from URL
